src/main/java/net/kodehawa/mantarobot/commands/CustomCmds.java

```diff
@@ -502,13 +499,13 @@ protected void call(Context ctx, String content) {
 
                 CustomCommand command = ctx.db().getCustomCommand(ctx.getGuild(), content);
                 String owner = command.getData().getOwner();
-                User user = owner.isEmpty() ? null : ctx.getShardManager().getUserCache().getElementById(owner);
+                Member member = owner.isEmpty() ? null : ctx.getGuild().retrieveMemberById(owner, false).complete();
```

Comment:
what about owners that leave? this will throw on those cases

Reply:
Very fixed